float64[] tau_c
float64[] w_c
float64[] f_c
float64[] t_c

float64[] f_meas
float64[] t_meas

string contact_frame
